jena-db/jena-tdb2/src/main/java/org/apache/jena/tdb2/xloader/ProcIndexBuildX.java

```diff
@@ -141,6 +141,7 @@ private static long sort_build_index(Logger LOG, String datafile, DatasetGraph d
             return 0;
         // Sort task.
         Process proc2;
+        ProcessBuilder pb2;
```

Comment:
This can be inside the try loop - only proc2 needs to be passed out so the process can be stopped. Once built, the ProcessBuilder isn't needed.

```
ProcessBuilder procBuilder = new ProcessBuilder(sortCmd);
procBuilder.environment().put("LC_ALL","C");
proc2 = procBuilder.start();
```

Reply:
Thanks for the comment @afs, moved inside the try block.